dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilderTest.java

```diff
@@ -208,25 +208,15 @@ public void testBuildVerboseDependencyGraph_singleArtifact() {
         dependencyGraphBuilder.buildVerboseDependencyGraph(logging);
 
     List<DependencyPath> list = graph.list();
-    Assert.assertEquals(
-        "commons-logging",
-        list.get(0).getLeaf().getArtifactId());
-    Assert.assertEquals(
-        "log4j",
-        list.get(1).getLeaf().getArtifactId());
-    Assert.assertEquals(
-        "logkit",
-        list.get(2).getLeaf().getArtifactId());
-    Assert.assertEquals(
-        "avalon-framework",
-        list.get(3).getLeaf().getArtifactId());
-    Assert.assertEquals(
-        "servlet-api",
-        list.get(4).getLeaf().getArtifactId());
-    Assert.assertEquals(
-        "javaee-api",
-        list.get(5).getLeaf().getArtifactId());
-    Assert.assertEquals(6, list.size()); // optional dependencies are included
+
+    // The list should not include provided dependencies
+    Truth.assertThat(list)
+        .comparingElementsUsing(
+            Correspondence.transforming(
```

Comment:
The Correspondence is a good candidate for extracting into a constant, so this can read

```
assertThat(list)
    .comparingElementsUsing(LEAF_ARTIFACT_ID)
    .containsExactly(...)
    .inOrder();
```

Reply:
Updated to use LEAF_ARTIFACT_ID.